sparse/bm25/bm25-vector-generation.ipynb

```diff
@@ -0,0 +1,430 @@
+{
```

Comment:
<p>This notebook should not upsert to Pinecone at all, only show how to generate the data</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pinecone-io/examples/pull/132/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
What? People need to know how to use this product in a concise example